cmd/juju/application/deploy.go

```diff
@@ -433,39 +438,32 @@ when refreshing the application in the future.
 A local charm may be deployed by giving the path to its directory:
 
   juju deploy /path/to/charm
-  juju deploy /path/to/charm --series bionic
+  juju deploy /path/to/charm ---base ubuntu@22.04
 
 You will need to be explicit if there is an ambiguity between a local and a
 remote charm:
 
   juju deploy ./pig
   juju deploy ch:pig
 
-An error is emitted if the determined series is not supported by the charm. Use
-the '--force' option to override this check:
-
-  juju deploy charm --series bionic --force
-
-A bundle can be expressed similarly to a charm:
+  A bundle can be expressed similarly to a charm:
 
   juju deploy mediawiki-single
-  juju deploy mediawiki-single --series focal
+  juju deploy mediawiki-single --base ubuntu@22.04
   juju deploy ch:mediawiki-single
 
 A local bundle may be deployed by specifying the path to its YAML file:
 
   juju deploy /path/to/bundle.yaml
 
-The final charm/machine series is determined using an order of precedence (most
+The final charm/machine base is determined using an order of precedence (most
 preferred to least):
 
- - the '--series' command option
- - the series stated in the charm URL
+ - the '--base' command option
  - for a bundle, the series stated in each charm URL (in the bundle file)
  - for a bundle, the series given at the top level (in the bundle file)
- - the 'default-series' model key
- - the top-most series specified in the charm's metadata file
-   (this sets the charm's 'preferred series' in the Charm Store)
+ - the 'default-base' model key
+ - the top-most base specified in the charm's metadata file
```

Comment:
a base isn't specified in the metadata file, it's in the manifest. And the "top-most" no longer makes sense as it's not charm author definable. Working on a story now. I suppose that changing the filename to manifest here is the best at the moment.

Reply:
Agreed.